### DISCUSSION TITLE
Does eFluor 615 dye inactivate?

### DISCUSSION THREAD

**Discussion by lamborni** [IBEXImagingCommunity/ibex_imaging_knowledge_base#59] [Q&A] [answered]
Asking for friend...

**Comment by radtkea** [comment, marked as answer]
Haha...aren't we all asking for a friend?

Thanks so much for your question! eFluor 615 is great for multiplexed 2D imaging but the signal is still present after prolonged exposure (120+ minutes) of 1 mg/ml LiBH4. It behaves very similar to Alexa Fluor 594 and can be placed in the last cycle of an IBEX experiment or used as a fiducial.

To find the inactivation conditions of a particular fluorophore/dye, please go to the Reagent Resources table as shown in the attached picture or go to this [link](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html) that displays all the fluorescent probes tested by the IBEX Imaging Community. 

![IBEX_Conjugate](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/67d00b66-5a9e-414e-88e0-d235f27c8869)
![Fluorescentprobes](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/81cc2b56-bcbb-4bc2-a5c8-a64e0f2ee084)

Best of luck!
A
